skglm/datafits/group_task.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from numpy.linalg import norm
+from numba.experimental import jitclass
+from numba import int32, float64
+
+from skglm.datafits.base import BaseDatafit
+
+
+spec_QuadraticGroup = [
+    ('grp_partition', int32[:]),
```

Comment:
use `grp_ptr` to stay coherent with the scipy sparse format this is inspired from. ptr stands for "pointer" not "partition"